gadopt/stokes_integrators.py

```diff
@@ -814,19 +800,19 @@ def __init__(
         displacement: fd.Function,
         **kwargs,
     ) -> None:
-        super().__init__(solution, approximation, **kwargs)
+        if "dt" not in kwargs:
+            raise TypeError(f"A timestep (the 'dt' keyword argument) must be provided "
```

Comment:
I think this is definitely better than we had, but should we make dt a required argument of the Viscoelastic solver?